client/components/Login.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export default function Login() {
+  return (
+    <div>
+      <h1 className="login">Login</h1>
+      <form className="form" action="#">
+        Username
+        <input type="text" name="username" />
```

Comment:
Maybe have a space between Username and the input field? Same with the password field